### PR TITLE
Add Latinization

### PR DESCRIPTION
**Pull request by EthanSchatzline**

### CODE REVIEW

**Comment by EthanSchatzline** [comment]
@ArtSabintsev I figured out latinization by using CFStringTransform, which can also remove the accents so I changed the withoutAccents function to do the same, and then the latinization function does its thing and then calls withoutAccents at the end

**Comment by EthanSchatzline** [comment]
I was doing it with CFStringTransform, but since ios9, they have `String.applyingTransform()` so I changed it to that

File: Sources/GuitarCharacter.swift [hunk 1]
```diff
@@ -36,6 +36,18 @@ public extension String {
...
+    /// Returns the latinized version of the string without diacritics.
+    ///
+    ///     let string = "Hello! こんにちは! สวัสดี! مرحبا! 您好!"
+    ///     print(string.latinized())
+    ///     // Prints "Hello! kon'nichiha! swasdi! mrhba! nin hao!"
+    ///
+    /// - Returns: The latinized version of the string without diacritics.
+    @discardableResult
+    func latinized() -> String {
+        return (applyingTransform(.toLatin, reverse: false) ?? self).withoutAccents()
```
Comment (EthanSchatzline) [inline Sources/GuitarCharacter.swift:48]: `String.applyingTransform` returns an optional, so I had to use `??` and just defaulted to returning self if for some reason it's nil

**Comment by ArtSabintsev** [comment]
Looks good - may make some changes after the merge, but overall, great job!

**Pull request merged**